tests/unit/src/main/scala/tests/BaseWorksheetLspSuite.scala

```diff
@@ -5,323 +5,335 @@ import scala.meta.internal.metals.UserConfiguration
 import scala.meta.internal.metals.MetalsSlowTaskResult
 import scala.concurrent.Promise
 
-abstract class BaseWorksheetLspSuite(scalaVersion: String)
+abstract class BaseWorksheetLspSuite(scalaVersions: Set[String])
```

Comment:
For consistency with the rest of the codebase, I would prefer to keep the old signature and duplicate instead subclasses.

Reply:
I'll revert that.